packages/vapor/scss/typography/elements.scss

```diff
@@ -3,3 +3,36 @@
     font-size: var(--default-font-size);
     line-height: $big-line-height;
 }
+
+.text {
+    --color: var(--default-text-color);
+    color: var(--color);
+
+    &.mod-emphasized {
```

Comment:
Do we have some examples in React-Vapor to see the result? (or these classes are more relevant in the admin-ui?)

Reply:
I did mention to Rosalie-Senpai this morning that we should have a demo for everything related to text (font, colors, sizes, use cases, etc).

Food for thoughts :smile_cat:

Reply:
Maybe in the style section :thinking: 
@j-mcgregor Tooling Bruv, what do you think? :eyes:

Reply:
I added examples for Text colors and Icon colors, and more improvements can be done in other user stories.
![image](https://user-images.githubusercontent.com/52677246/110539917-d55d4580-80f3-11eb-97a1-810e854d2e6b.png)
![image](https://user-images.githubusercontent.com/52677246/110539950-dee6ad80-80f3-11eb-85e3-0399cd9acfea.png)